### DISCUSSION TITLE
Mouse middle click support on the "Find ..." menu items

### DISCUSSION THREAD

**Discussion by ViacheslavPrytykin** [datalust/seq-tickets#1157] [Ideas]
Often I want to keep the current search results and make a subsequent search in a new tab by either event type or some event parameter.
Currently, if I go to "Event -> Find" or "✔ ❌ RequestId -> Find" and then click on "Find" with a mouse middle button click, it opens a new Seq tab, but the search field does not contain "@EventType = 0x..." or "@RequestId = ...". It's very frustrating.

**Comment by nblumhardt** [comment]
Thanks for the suggestion; we've discussed this in the past and not found a nice clean way to implement it, but we're revisiting it now and will let you know what we come  up with.

We're using discussions to track unscheduled enhancements and ideas, so I'll move this over there now. If we are able to move it forwards, we'll reopen the issue.

Thanks again!
